frontend/pages/beta/model/new.tsx

```diff
@@ -151,9 +153,16 @@ export default function NewModel() {
               <Box sx={{ textAlign: 'right' }}>
                 <Tooltip title={!formValid ? 'Please make sure all required fields are filled out' : ''}>
                   <span>
-                    <Button variant='contained' disabled={!formValid} type='submit' data-test='createModelButton'>
+                    <LoadingButton
+                      variant='contained'
+                      disabled={!formValid}
+                      type='submit'
+                      data-test='createModelButton'
+                      onClick={onSubmit}
```

Comment:
You shouldn't need this `onSubmit` here, because the form has an `onSubmit` call.